src/Xamarin.Android.Build.Tasks/Utilities/ManagedResourceParser.cs

```diff
@@ -124,61 +124,64 @@ public CodeTypeDeclaration Parse (string resourceDirectory, IEnumerable<string>
 
 		void ProcessRtxtFile (string file)
 		{
+			Console.WriteLine (file);
 			var lines = System.IO.File.ReadLines (file);
 			foreach (var line in lines) {
+				Console.WriteLine (line);
 				var items = line.Split (new char [] { ' ' }, 4);
 				int value = items[0] != "int[]" ? Convert.ToInt32 (items [3], 16) : 0;
 				string itemName = items [2];
 				switch (items [1]) {
-					case "anim":
-						CreateIntField (animation, itemName, value);
+				case "anim":
```

Comment:
Do we need this whitespace change? (Are your IDE settings screwing things up again? ;-)

Reply:
I was fixing the formatting.. the indentation of the case's was incorrect.